charts/spire/templates/agent-daemonset.yaml

```
@@ -31,7 +31,7 @@ spec:
           # from https://github.com/lqhl/wait-for-it
```

Please update location: https://github.com/vishnubob/wait-for-it
```suggestion
          # from https://github.com/vishnubob/wait-for-it
```